log/README.md

```diff
@@ -0,0 +1,54 @@
+# package log
+
+`package log` is an opinionated package for logging in your service.
```

Comment:
Thoughts on removing:
> is an opinionated package for logging in your service.

Consider:

`package log` provides a minimal interface for structured logging, which may be wrapped to encode conventions, enforce type-safety, etc.